src/tools/run-make-support/src/llvm.rs

```diff
@@ -61,6 +61,22 @@ impl LlvmReadobj {
         Self { cmd }
     }
 
+    /// Specify the format of the ELF information.
+    ///
+    /// Valid options are `LLVM` (default), `GNU`, and `JSON`.
+    pub fn elf_output_style(&mut self, style: &str) -> &mut Self {
+        self.cmd.arg("--elf-output-style");
+        self.cmd.arg(style);
+        self
+    }
+
+    /// Set the formation of the ELF information as `GNU`.
+    ///
+    /// Convenience function for [`elf_output_style`].
+    pub fn gnu_elf_style(&mut self) -> &mut Self {
```

Comment:
This is implemented to maintain compatibility with the output of GNU `readelf`, although the LLVM output may be sufficient. AFAIK, the LLVM format adds additional information before the section content (file name, format, architecture, loadname), which could possibly mess with scanning stdout.

Reply:
This requires the `gnu_elf_style` to be called for every invocation of `readelf` to maintain compatibility, which is not ideal. Setting GNU as the default style would help and not affect the non-ELF uses.

Reply:
I think it's fine if we set gnu as the default elf style. If we run into any tests that expect different elf styles, we could always introduce an alternative constructor that does not set gnu as default elf style.